{{cookiecutter.project_slug}}/docker-compose.shared.admin.yml

```diff
@@ -1,6 +1,6 @@
 version: '3.3'
 services:
   proxy:
-    image: traefik:v1.6
+    image: traefik:v1.7.9-alpine
```

Comment:
Can you please make this `traefik:v1.7`?

Reply:
done